includes/discovery/sensors/state/quanta.inc.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/*
+ * LibreNMS Quanta LB6M States information module
+ *
+ * Copyright (c) 2017 Mark Guzman <segfault@hasno.info>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+d_echo('Quanta Chassis state');
+//FASTPATH-BOXSERVICES-PRIVATE-MIB::boxServicesTempUnitState
+$chassis_state_oid = '.1.3.6.1.4.1.4413.1.1.43.1.15.1.2.1';
+
+$state_name = 'quanta_state';
+$descr = 'Chassis state';
+$state = snmp_get($device, $chassis_state_oid, '-Oqv');
+if (!empty($state)) {
+    $state_index_id = create_state_index($state_name);
+    if ($state_index_id !== null) {
+        $states = array(
+            array($state_index_id, 'other', 0, 0, 3),
+            array($state_index_id, 'ok', 1, 1, 0),
+            array($state_index_id, 'degraded', 1, 2, 1),
+            array($state_index_id, 'failed', 1, 3, 2),
+        );
+
+        foreach ($states as $value) {
+            $insert = array(
+                'state_index_id' => $value[0],
+                'state_descr' => $value[1],
+                'state_draw_graph' => $value[2],
+                'state_value' => $value[3],
+                'state_generic_value' => $value[4]
+                );
+            dbInsert($insert, 'state_translations');
+        }
+    }
+    discover_sensor($valid['sensor'], 'state', $device, $chassis_state_oid, 1, $state_name, $descr, '1', '1', null, null, null, null, $state, 'snmp', 1);
```

Comment:
Swap the 1 you've used for index to `boxServicesTempUnitState` on this line and the one below.